reflekt-dsl/src/main/kotlin/org/jetbrains/reflekt/ReflektClass.kt

```diff
@@ -0,0 +1,44 @@
+package org.jetbrains.reflekt
+
+import kotlin.reflect.*
+
+/**
+ * Represents a class and designed for storing information about it, which is collected by Reflekt at compile-time.
+ *
+ * @param T the type of the class.
+ * @property kotlin [KClass] instance for this class.
+ * @property annotations Annotations which are present on this class.
+ * @property isAbstract `true` if this class is `abstract`.
+ * @property isCompanion `true` if this class is a companion object.
+ * @property isData `true` if this class is a data class.
+ * @property isFinal `true` if this class is `final`.
+ * @property isFun `true` if this class is a Kotlin functional interface.
+ * @property isInner `true` if this class is an inner class.
+ * @property isOpen `true` if this class is `open`.
+ * @property isSealed `true` if this class is `sealed`.
+ * @property isValue `true` if this class is a value class.
+ * @property qualifiedName The fully qualified dot-separated name of the class, or `null` if the class is local or a class of an anonymous object.
+ * @property superclasses The set of immediate superclasses of this class.
+ * @property sealedSubclasses The set of the immediate subclasses if this class is a sealed class, or an empty set otherwise.
+ * @property simpleName The simple name of the class as it was declared in the source code,
+ * or `null` if the class has no name (if, for example, it is a class of an anonymous object).
+ * @property visibility Visibility of this class, or `null` if its visibility cannot be represented in Kotlin.
+ */
+data class ReflektClass<T : Any>(
```

Comment:
What is about functions? I mean e.g. receivers

Reply:
In this PR I am going only to have info for classes.